cypress/e2e/dashboard/events/index.cy.js

```diff
@@ -16,10 +18,15 @@ describe('events dashboard', () => {
     cy.exec('npm run db:seed');
     cy.login();
     cy.mhDeleteAll();
+    cy.intercept('POST', 'http://localhost:5000/graphql', (req) => {
+      // Queries
+      aliasQuery(req, 'events');
+    });
```

Comment:
```suggestion
    cy.interceptGQL('events');
```
It's a bit DRYer if we reuse the existing command.  Also I don't think we need to distinguish between queries and mutations here.  It should be clear enough from context.